guides/doc-Administering_Red_Hat_Satellite/topics/proc_integrating-satellite-with-red-hat-single-sign-on-for-external-authentication.adoc

```diff
@@ -218,12 +217,12 @@ For more information about group mappers, see https://access.redhat.com/document
 +
 . Open the `_{Keycloak-short}.example.com_/auth/realms/_{Keycloak-short}_REALM_/.well-known/openid-configuration` URL and note the values to populate the options in the following steps.
 +
-. Set the value for the Open IDC audience:
+. Add the value for the Hammer client in the Open IDC audience. Note that if you have already registered another client with keycloak, make sure also to append that audience in the array. For example:
```

Comment:
I would keep the earlier version  and add this as a separate note below.

Reply:
Ack, done.